frontend/vue.config.js

```diff
@@ -28,6 +30,17 @@ module.exports = {
       .end()
       .use('vue-svg-loader')
       .loader('vue-svg-loader')
+
+    config.module
+      .rule('source-map-loader')
+      .test(/\.(js)$/)
```

Comment:
https://github.com/ecamp/hal-json-vuex/pull/9 Typescript is about to land in hal-json-vuex, so maybe add `ts` to the file types? Or is that not necessary since it will be compiled to plain js anyways?

Reply:
I think that's not necessary since, as you mentioned, we have the compiled JS in the dependencies. This is the recommended configuration for this loader: https://webpack-v3.jsx.app/loaders/source-map-loader/.